docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -22,6 +22,7 @@ instances.
   [OpenEBS](openebs-operator.md) or one of the
   [built-in](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes)
   plugins.
+* The [cert-manager component](cert-manager.md) is required for enabling HTTPS.
```

Comment:
Actually, Ingress controller is also required for both and it is not mentioned :|

Reply:
Yeah, I actually had to setup contour, metallb, external-dns and cert-manager.  For some reason the other three were clear to me that were needed, the cert-manager one wasn't clear (I thought it would just work).  Should I add links to all 4 of them here?

Reply:
Ideally that complexity should be hide from the user, but we are not there yet.

I think having `contour` and `cert-manager` here is OK. `contour` should mention the requirement of `metallb` then and `external-dns` is always optional, as user may create DNS entries on their own, manually too. Perhaps we should document that, if it's not there yet.

Reply:
Alright, I've added a bullet point for contour, taken from the external-dns doc.

Reply:
Thanks! LGTM.